vm/2.0/modules/ROOT/pages/index.adoc

```diff
@@ -10,7 +10,7 @@ Anypoint Connector for Virtual Machine (VM Connector) manages intra-app and inte
 +
 When running a Mule application in single runtime instance mode, persistent queues work by serializing and storing the contents on the disk. However, when running the Mule application in cluster runtime instance mode, the persistent queues are backed up in the memory grid. Thereby, when a flow uses VM Connector to publish content to a queue, Mule runtime engine determines whether to process the message in the same origin node or to send the message to the cluster to pick another node. Using VM Connector facilitates the load distribution across a cluster. 
 +
-In either single or cluster runtime instance mode, when using persistent queues, the data you send must be serializable. Although you can enable Kryo serialization to get a wider range of serializable values, Kryo does have some limitations.
+In either single or cluster runtime instance mode, when using persistent queues, the data you send must be serializable. Although you can enable Kryo serialization to get a wider range of serializable values, Kryo does have some limitations. Refer to xref:mule-runtime::configure-custom-serializers.adoc#considerations-when-using-serialization[] for details.
```

Comment:
i believe you need to include a title in the [] or else this link appears as text